apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -94,7 +94,12 @@ export default class MicroBitBoard extends EventEmitter {
             // Log if we were not able to determine the firmware version in time.
             firehoseClient.putRecord({
               study: MAKER_TOOLKIT,
-              study_group: MICROBIT,
+              // TODO @snickell ESM: @fisher-alice this is in `staging` as `study_group: MICROBIT`, but
+              // that value isn't found in MicroBitConstants.js, even in history. I think this was
+              // probably a dangling already invalid reference that our old import checker couldn't catch.
+              // I've set it to `MB_API`, which has the value `microbit` and occurs close
+              // to FIRMWARE_VERSION_TIMEOUT in the same file. This was my best guess... please confirm! 🙏
+              study_group: MB_API,
```

Comment:
@fisher-alice does `MicroBitConstants.MB_API` (who's value is `"microbit"`, so `study_group: "microbit"`) seem like the right value to pass in as `study_group` here? In `staging` its current set to `MicroBitConstants.MICROBIT` which is a missing reference. I think our reference checker couldn't find these until we started using new-style imports.

Reply:
From Alice, `MB_API` is the correct and intended value here